CHANGELOG.md

```diff
@@ -10,13 +10,18 @@
 - enhancement - Show status bar text based on component analysis status. See [#459](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/459)
     - Notification will be shown only once per manifest in single session.
     - Further changes will be updated only via status bar.
-- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.4.15](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.4.15)): See [#459](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/459)
+- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.4.19](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.4.19)): See [#469](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/469)
     - Golang CA support from LSP. See [#148](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/148)
```

Comment:
Add the following under lsp,

https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/176
https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/177
https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/172